astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -70,7 +70,7 @@ def check_fn(files: List[Any]) -> bool:
         :return: true if the criteria is met
         :rtype: bool
         """
-        return all(f.get("Size", 0) > 1024 for f in files)
+        return all(f.get("Size", 0) > 20 for f in files)
```

Comment:
Can we update the docstring then?

Reply:
I was thinking if we should change it to `f.get("Size", 0) > 0`. This way it might be more stable and we are also testing the check function wdyt?

Reply:
yes, was going to suggest > 0, but thought you want to intentionally keep it to that size based on the files we want to test :)